FreeRTOS/FreeRTOS-Plus-TCP
# [IPv6] Verify options order in IPv6 Extension Headers

Description
-----------
This PR updates the verification of order of extension headers appearing in the IPv6 header as per RFC 2460 section [4.1](https://www.rfc-editor.org/rfc/rfc2460#section-4.1)  Extension Header Order:

>  IPv6 nodes must accept and attempt to process extension headers in
   any order and occurring any number of times in the same packet,
   except for the Hop-by-Hop Options header which is restricted to
   appear immediately after an IPv6 header only.


Test Steps
-----------
Protocol tests

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
